src/liveliness.rs

```diff
@@ -208,7 +197,6 @@ pub extern "C" fn z_liveliness_declare_subscriber(
     }
 }
 
-/// @warning This API has been marked as unstable: it works as advertised, but it may be changed in a future release.
```

Comment:
zc_liveliness_declare_background_subscriber is still not available in pico. I suggest we either rename it into z_... or leave under stable to keep the possibility to rename it into z_... in the future.

Reply:
fixed